app/models/subscription.rb

```diff
@@ -38,11 +38,18 @@ def name
   end
 
   def subject_text(creation)
-    authors = creation.pseuds.map{ |p| p.byline }.to_sentence
+    authors = if creation.is_a?(Work) && creation.anonymous? || creation.is_a?(Chapter) && creation.work.anonymous?
```

Comment:
You can use `anonymous_creation?` here.

Reply:
This was annoying the daylights out of me. I tried `anonymous_creation?(creation)` and `self.anonymous_creation?(creation)` and couldn't get it to work even though I _knew_ there had to be something. But now I see the problem! 🤦‍♂

Reply:
I'd prefer `self.class.anonymous_creation?(creation)`. `self.class` allows subclassing and less work when renaming the class (though neither are likely in this case).